app/src/main/res/layout-night/activity_auth.xml

```diff
@@ -106,7 +106,7 @@
 
```

Comment:
There is no need to use redundant View and FrameLayout. You can place all the elements using constraints and biases in a ConstraintLayout, sometimes adding a GuideLine to precisely constrain your view.